src/components/ProblemsTable.tsx

```diff
@@ -156,6 +187,21 @@ export const ProblemsTable = ({
           Filter: tableFilters.TextFilter(),
           minWidth: 50,
           maxWidth: 250,
+          Cell: (row: CellProps<Report>) => {
+            return (
+              <div
+                className='iui-anchor'
```

Comment:
Please avoid using internal iui- classes. You can use the `<Anchor>` react component.